lib/MCSClient.js

```diff
@@ -350,6 +352,39 @@ class MCSClient extends MCSBaseClient {
     }
   }
 
+  /**
+   *
+   * @param {String} roomId
+   * @param {String} mediaId
+   */
+  releaseContentFloor (roomId, mediaId) {
+    if (!roomId || typeof(roomId) !== 'string') {
+      throw new Error('Invelid userId');
```

Comment:
You're throwing nonsense errors, please fix them to reflect the actual parameters you're testing. This goes for all the messages you added in this PR.